packages/suite/src/constants/suite/device.ts

```diff
@@ -44,3 +44,11 @@ export const CHARACTER_OFFSET_FOR_NEXT_ARROW: Record<DeviceModelInternal, number
     [DeviceModelInternal.T3B1]: 2,
     [DeviceModelInternal.T3T1]: 4,
 };
+
+export const SUPPORTS_DEVICE_AUTH: Record<DeviceModelInternal, boolean> = {
+    [DeviceModelInternal.T1B1]: false,
+    [DeviceModelInternal.T2T1]: false,
+    [DeviceModelInternal.T2B1]: true,
+    [DeviceModelInternal.T3B1]: true,
+    [DeviceModelInternal.T3T1]: true,
+};
```

Comment:
nitpick: just maybe naming could be better. isn't `SUPPORTS_DEVICE_AUTH`. too generic? meaning that there might be a device that supports firmware revision check but doesn't not support firmware hash checek.

Reply:
https://github.com/trezor/trezor-suite/pull/13495/commits/8ff596a3791bd5e8ea78836d9ad10fed3135bd79